<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibration</class>
 <widget class="QMainWindow" name="Calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2060</width>
    <height>1328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabImg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>960</width>
      <height>770</height>
     </rect>
    </property>
    <widget class="QWidget" name="realTimeDisplay">
     <attribute name="title">
      <string>timestamp</string>
     </attribute>
     <widget class="QLabel" name="lblRealtimeImg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>960</width>
        <height>720</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabPointCloud">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>100</y>
      <width>960</width>
      <height>770</height>
     </rect>
    </property>
    <widget class="QWidget" name="realTimeDisplay_2">
     <attribute name="title">
      <string>timestamp</string>
     </attribute>
     <widget class="QVTKWidget" name="qvtkRealtimePointcloud" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>960</width>
        <height>720</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCalibrationMode">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>900</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>CalibrationMode</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>241</width>
       <height>84</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rdBtnCamOnly">
        <property name="text">
         <string>camera only</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnCamVel">
        <property name="text">
         <string>camera velodyne</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>900</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>CalibrationBoardSize</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>281</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxCalibrationBoardWidth">
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxCalibrationboardHeight">
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>square size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxCalibrationboardSquareSize">
        <property name="value">
         <double>35.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>1090</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>CalibrationBoardType</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>181</width>
       <height>84</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rdBtnChessBoard">
        <property name="text">
         <string>ChessBoard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdBtnCircleGrid">
        <property name="text">
         <string>CircleGrid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1810</x>
      <y>930</y>
      <width>144</width>
      <height>148</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="btnCapture">
       <property name="text">
        <string>Capture</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCalibration">
       <property name="text">
        <string>Calibration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>1160</y>
      <width>258</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonFixAspectRatio">
       <property name="text">
        <string>FIX ASPECT RATIO</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAspectRatio">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2060</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
